src/components/ui/icons/TrashIcon/index.jsx

```diff
@@ -0,0 +1,29 @@
+import "@components/ui/icons/_iconBase.scss";
+import PropTypes from "prop-types";
+
+function TrashIcon({ className = "icon-base", strokeWidth = 1.5, ...rest }) {
+    return (
+        <svg
```

Comment:
can't we use image in this case?

Reply:
I was thinking about the scalability. So that, I've used SVG here... Please suggest me what is the best practice. Thanks!